base-notebook/Dockerfile

```diff
@@ -113,7 +113,7 @@ RUN wget --quiet "https://github.com/conda-forge/miniforge/releases/download/${m
     conda config --system --set auto_update_conda false && \
     conda config --system --set show_channel_urls true && \
     if [ ! $PYTHON_VERSION = 'default' ]; then conda install --yes python=$PYTHON_VERSION; fi && \
-    conda list python | grep '^python ' | tr -s ' ' | cut -d '.' -f 1,2 | sed 's/$/.*/' >> $CONDA_DIR/conda-meta/pinned && \
+    conda list python | grep '^python ' | tr -s ' ' | cut -d ' ' -f 1,2 >> $CONDA_DIR/conda-meta/pinned && \
```

Comment:
This is too technical for me to parse, can you describe what is changed in what was previously written to the pinned file vs what is now written to the pinned file?

Reply:
For example, the current version of python is `3.9.2`.
Currently, in the master branch we will pin the version like this `python 3.9.*`.
So, during conda update (and even install), we might update (and even downgrade) python version to `3.9.x`.

The change: write exactly `python 3.9.2`. This way we pin python versions in a strict way and it won't change during conda update / install.

Reply:
Excellent thank you!